src/tabs/tab.jsx

```diff
@@ -118,12 +125,29 @@ const Tab = React.createClass({
       width: width,
     }, style);
 
+    let iconElement;
+    if (icon) {
+      const params = {
+        style: {
+          fontSize: 24,
+          display: label ? 'block' : 'inline-block',
+          color: selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor,
```

Comment:
Can we use a common variable for `selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor`?